romana-install/roles/romana/postinstall/kubernetes/master/tasks/main.yml

```diff
@@ -14,10 +14,5 @@
   become: true
   become_user: root
 
-- name: Install romana tool
-  template: src="../cmd/romana.k8s" dest="{{ romana_bin_dir }}/romana" mode=0775
```

Comment:
I'm pleased that the config is a template, not the script/command itself.
Almost like a real company's CLI tool. (Did I get that reference right?)